samples/spring-valueentity-counter/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+
+version: "3"
+services:
+  kalix-proxy:
+    image: gcr.io/kalix-public/kalix-proxy:1.0.22
```

Comment:
```suggestion
    image: gcr.io/kalix-public/kalix-proxy:1.0.24
```

Reply:
Other samples are on 1.0.22 as well. We usually wait until the proxy version reaches production.